public/avf_graphs/area.chart.js

```diff
@@ -0,0 +1,427 @@
+import { GraphLayout } from "./graph.layout.js";
```

Comment:
The number of js files is increasing and having non-descriptive folder names for them will make it hard to maintain. Please; 
- [ ] store/move all js files in one `js` folder.
- [ ] Rename all js files with respect to the class name e.g UI.js -> UIController.js so that we don't have to remember 2 names.
- [ ] Move css file to CSS folder.

Reply:
Updated thanks.